File: atv-topico-3-4/topico-4-js/splitints.js
const splitints = (a) => {
  console.log(
    a.filter((e) => e % 2 != 0),
    a.filter((e) => e % 2 == 0)
  );
};

splitints([1, 2, 3, 4, 5, 6, 7, 8, 9, 10]);

File: atv-topico-3-4/topico-4/splitints.hs
splitints :: Integral a => [a] -> ([a], [a])
splitints u = (filter (\x -> x `mod` 2 /= 0) (u) , filter (\x -> x `mod` 2 == 0) (u))

main = do
    a <- readLn :: IO [Int]
    print $ splitints a